src/main/java/uk/gov/companieshouse/insolvency/delta/consumer/InsolvencyDeltaConsumer.java

```diff
@@ -26,21 +26,23 @@ public InsolvencyDeltaConsumer(InsolvencyDeltaProcessor deltaProcessor) {
     /**
      * Receives Main topic messages.
      */
-    @KafkaListener(topics = "${insolvency.delta.topic.main}", groupId = "insolvency.delta.topic.main")
+    @KafkaListener(topics = "${insolvency.delta.topic.main}", groupId = "insolvency-delta-consumer")
     @Retryable
     public void receiveMainMessages(Message<ChsDelta> chsDeltaMessage) {
-        LOGGER.info("A new message read from MAIN topic with payload: " + chsDeltaMessage.getPayload());
+        LOGGER.info("A new message read from MAIN topic with payload: "
+                + chsDeltaMessage.getPayload());
         deltaProcessor.processDelta(chsDeltaMessage);
     }
 
     /**
      * Receives Retry topic messages.
      */
-    @KafkaListener(topics = "${insolvency.delta.topic.retry}", groupId = "insolvency.delta.topic.retry")
```

Comment:
Still should think about this, do we really want two listeners/consumers? 
Could either: 
-have 1 and pass topic as an envar
-have 2 but envvar bool to say if retry and only run 1 based on this value
-all instances consume from both

Decision can probably be left for now

Reply:
Agree, lets discuss and improve as part of consumer stories